llvm/test/Transforms/PGOProfile/unreachable-block.ll

```diff
@@ -0,0 +1,13 @@
+; RUN: llvm-profdata merge %S/Inputs/unreachable-block.proftext -o %t.profdata
```

Comment:
Please use `split-file` so that the `.proftext` file is inline in this test

https://llvm.org/docs/TestingGuide.html#extra-files